cmd/krel/cmd/release_notes.go

```diff
@@ -764,7 +668,7 @@ func releaseNotesJSON(tag string) (string, error) {
 	}
 
 	logrus.Infof("Using start tag %v", startTag)
-	logrus.Infof("Using end tag %v", tag)
+	logrus.Infof("Using end tag   %v", tag)
```

Comment:
Is this change intentional? If not, then I suggest:
```suggestion
	logrus.Infof("Using end tag %v", tag)
```